de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/functions/getBPOpenAmtToDate.sql

```diff
@@ -22,7 +22,7 @@ BEGIN
                                        p_DateType := (
```

Comment:
does `invoiceOpenToDate()` properly deal with credit memos?

Reply:
If you refer to negating the amount in case the docbasetype is of credit memo, then no, it does not.

I don't think we should fix that function in this issue, i think we should make a new issue for the fix. We should also create a real example and set what we expect and what should and shouldn't happen.